AnnoyancesFilter/sections/popups.txt

```diff
@@ -97,6 +97,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+studwork.org#$#.base-modal { display: none !important; }
+studwork.org#$#.base-modal__shadow { display: none !important; }
+studwork.org#$#body { overflow: auto !important; }
```

Comment:
studwork.org#$#body { overflow: visible !important; padding-right: 0 !important; }

Reply:
It does the same work. So what's the difference?

Reply:
And I don't find `padding-right: 0` anywhere.

Reply:
> So what's the difference?

You can read it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow). I don't see the point in using `auto` here.

> And I don't find `padding-right: 0` anywhere.

When a modal window appears, not only the `overflow` property changes, but also the `padding-right` property.